.aws/task-definition.json

```diff
@@ -21,33 +21,53 @@
       "logConfiguration": {
         "logDriver": "awslogs",
         "options": {
-          "awslogs-group": "awslogs-tis-template",
+          "awslogs-group": "awslogs-tis-revalidation-concerns",
           "awslogs-region": "eu-west-2",
-          "awslogs-stream-prefix": "awslogs-tis-template"
+          "awslogs-stream-prefix": "awslogs-tis-revalidation-concerns"
         }
       },
       "secrets": [
         {
-          "name": "DB_HOST",
-          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-template-db-host"
+          "name": "MONGODB_HOST",
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/reval-mongo-host"
         },
         {
-          "name": "DB_PORT",
-          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-template-db-port"
+          "name": "MONGODB_PORT",
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/reval-mongo-port"
         },
         {
-          "name": "DB_USER",
-          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-template-db-user"
+          "name": "MONGODB_USERNAME",
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/reval-mongo-user"
         },
         {
-          "name": "DB_PASSWORD",
-          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-template-db-password"
+          "name": "MONGODB_PASSWORD",
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/reval-mongo-password"
+        },
+        {
+          "name": "RABBITMQ_HOST",
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/reval-rabbit-host"
+        },
+        {
+          "name": "RABBITMQ_PASSWORD",
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/reval-rabbit-password"
+        },
+        {
+          "name": "RABBITMQ_PORT",
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/reval-rabbit-port"
+        },
+        {
+          "name": "RABBITMQ_USERNAME",
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/reval-rabbit-username"
+        },
+        {
+          "name": "TCS_URL",
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tcs-url"
```

Comment:
Do we need Rabbit and TCS for concerns?

Reply:
@Judge40 I have removed tcs and rabbit secrets